grafana/k6-docs
# Feat: replace wip-nit-icons

**Describe** what changes this pull request brings
This pull request changes icons in WIP & NIT components for Docs.

**Steps to test**
1. Open the [page]()
2. Confirm that everything looks and works as expected

<details>
<summary>Screenshots</summary>

WIP-icon
<img width="1641" alt="Screenshot 2022-09-09 at 12 59 54" src="https://user-images.githubusercontent.com/75138180/189301917-38742fc4-3527-44d6-b742-47619d8a46f5.png">

NIT-icon
<img width="1466" alt="Screenshot 2022-09-09 at 12 45 12" src="https://user-images.githubusercontent.com/75138180/189301621-6c5fc60a-68f0-4c02-be71-4a45e4be6913.png">


</details>

**References**
[Design](https://www.figma.com/file/qOPbJuR5wWjOablZeFCQtu/K6-Website-Design?node-id=10359%3A35637)

Closes:
